bedrock/firefox/views.py

```diff
@@ -528,5 +528,8 @@ class HelloStartView(LatestFxView):
     def get_context_data(self, **kwargs):
         ctx = super(HelloStartView, self).get_context_data(**kwargs)
         incoming = self.request.GET.get('incomingConversation') or 'none'
+        source = self.request.GET.get('utm_content') or 'none'
+        source = source.replace('hello-tour_OpenPanel_', '')
```

Comment:
> I still see utm_content=hello-tour_OpenPanel_firstrun in the URL for example.

The code isn't meant to change the URL. I suppose we could, but I don't really know the reason for it. This code parses the `utm_content` value and injects the last bit it into the template where it's used in the `data-tour-source` attribute. I'll throw a test in.